src/chunkserver/filesystem_adaptor/curve_filesystem_adaptor.cpp

```diff
@@ -22,17 +22,18 @@
 
 #include <butil/fd_utility.h>
 #include <vector>
+#include <list>
 
-#include "src/chunkserver/raftsnapshot/curve_filesystem_adaptor.h"
+#include "src/chunkserver/filesystem_adaptor/curve_filesystem_adaptor.h"
 
 namespace curve {
 namespace chunkserver {
 CurveFilesystemAdaptor::CurveFilesystemAdaptor(
-                                std::shared_ptr<FilePool> chunkFilePool,
+                                std::shared_ptr<FilePool> filePool,
                                 std::shared_ptr<LocalFileSystem> lfs) {
     lfs_ = lfs;
-    chunkFilePool_ = chunkFilePool;
-    uint64_t metapageSize = chunkFilePool->GetFilePoolOpt().metaPageSize;
+    filePool_ = filePool;
```

Comment:
Why did you change the name of chunkfilepool to filepool?

Reply:
because it can be walpool